.github/workflows/qa-viz.yml

```diff
@@ -3,31 +3,35 @@ name: QA Visualisation Workflow
 
 on:
   workflow_dispatch:
-    inputs: 
-      environment:
```

Comment:
Given this is removed, presumably because of the Redis distributed caching, do we see any risks with that in general, not just with this tool?

Reply:
Couldn't stay away? 😄 
The reason I've removed this is because the visualisation tool had to run a bit differently - rather than it using the database for a specific environment dev/test/staging/prod it spins up its own version of the app based off some branch and qa-tool gets questions from there. 
So it is based on branch + yes/no to preview content. I don't think theres a risk with the redis caches as those are separate per env so we won't get a case of preview content being saved and used in a non-preview env